DBM-GUI/localization.br.lua

```diff
@@ -39,13 +39,13 @@ L.MoveMe 					= "Mova-me"
 L.Button_OK 				= "OK"
 L.Button_Cancel 			= "Cancelar"
 L.Button_LoadMod 			= "Carregar AddOn"
-L.Mod_Enabled				= "Habilitar: %s"
+L.Mod_Enabled				= "Ativar: %s"
 L.Mod_Reset					= "Carregar opções padrão"
-L.Reset 					= "Resetar"
+L.Reset 					= "Redefinir"
 L.Import					= "Importar"
 
-L.Enable					= "Habilitar"
-L.Disable					= "Desabilitar"
+L.Enable					= ENABLE
+L.Disable					= DISABLE
```

Comment:
Since it's a global, doesn't need to be translated

Reply:
I just keep them in so people don't think they are missing in the future

Reply:
Even globals? that's seems wildly unnecessary. Why not comment the line out?